### DISCUSSION TITLE
Syntax highlight theme on Files tab

### DISCUSSION THREAD

**Discussion by VasilisManol** [extrawurst/gitui#1928] [Q&A]
Hi,

Is there a way to change the theme used in Files tab for syntax highlighting? It is unreadable on any terminal with light background. The rest of the GUI is rendered nicely on a light terminal. It is just the opened file under Files and only on light background that is very pale and unreadable.

**Comment by savchenko** [comment]
Created https://github.com/extrawurst/gitui/issues/2308

**Reply by extrawurst** [reply]
we rather need a way to style the syntax highlighting

**Reply by savchenko** [reply]
Of course, matter of choice between _"text looks fancy at some point in future"_ and _"text is readable now"_.
